### PR TITLE
RPM Specfile addition for intial RPM building process

### PR DESCRIPTION
**Pull request by iranzo**
Initial SPEC file for RPM creation of TSN Testbench

### CODE REVIEW

File: red hat/tsn-testbench.spec [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+cd %{name}-%{version}
+git checkout v%{version}
+
+%build
+cd %{_topdir}/BUILD/%{name}-%{version}
+cmake -DCMAKE_BUILD_TYPE=Release
+make DESTDIR=${RPM_BUILD_ROOT} install
+
+# Build documentation
+make -C Documentation   man
```
Comment (shifty91) [inline red hat/tsn-testbench.spec:39]: I don't think there's a man page yet?

Comment (iranzo) [reply red hat/tsn-testbench.spec:39]:
~~~
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... linuxtsntestbench.1 { introduction quickstart contribution build usage configuration mqtt test debug qa } done
build succeeded.

The manual pages are in _build/man.
~~~
image

and result

image

**Comment by shifty91** [comment]
Do you want this merged as this? This is labeled as draft.

**Comment by iranzo** [comment]
Hi Kurt, I left it as draft as I'm doing some testing with it, I would like to use extra vars for the release (now hardcoded to 5) with the goal to tie a new release with an automatic triggering of rpmbuild (both binary and srpm) for each release.

With the current SPEC I can get both rpm's created on my system and later install the binary rpm correctly.

If you're ok, we can merge and work in iterations over the remaining pieces

**Comment by shifty91** [comment]
> Hi Kurt, I left it as draft as I'm doing some testing with it, I would like to use extra vars for the release (now hardcoded to 5) with the goal to tie a new release with an automatic triggering of rpmbuild (both binary and srpm) for each release.
> 
> With the current SPEC I can get both rpm's created on my system and later install the binary rpm correctly.
> 
> If you're ok, we can merge and work in iterations over the remaining pieces

OK, I see. Then, I'll leave it open until you're testing is finished. Thanks!

**Comment by iranzo** [comment]
I did set some of the vars as variables, that need to be provided via rpmbuild CLI and I'll try to get github build the repo RPM  automatically via another PR

**Review by shifty91 — APPROVED** [review]

**Pull request closed without merging**